### DISCUSSION TITLE
Add the possibility to use attributes in the 'Home Page - Sensors'  section (a.o) of the blueprint

### DISCUSSION THREAD

**Discussion by kees-jans** [Blackymas/NSPanel_HA_Blueprint#1827] [Ideas]
After running into some issues, mostly due to very limited knowledge of HA, I managed to get my nspanel up and running and I like it a lot!! One thing that I didn't manage is displaying the attribute value of an entity on my home page (and entity pages). For example the target temperature of my Sonoff TRVZB or the lowest KWh price of electricity for this day (is an attribute of the 'electricity price' entity provided by the API of my provider. When I use an entity with attributes in an automation I can select an attribute from a pull-down menu.
Wouldn't it be nice if this could be added to the blueprint as well?
I have tried several ways to turn an attribute into an entity using the many examples in the HA groups but until now didn't manage to make it work (most likely again due to lack of knowledge). Suggestions for a simple alternative are more than welcome!

**Comment by edwardtfn** [comment]
- #1831
